### ISSUE TITLE
wrong axis label on speed up curves

### ISSUE THREAD

**Issue by ogrisel** [mbatoul/sklearn_benchmarks#12]
<img width="1258" alt="image" src="https://user-images.githubusercontent.com/89061/144077371-184bc969-fd11-45bc-8db8-b0798b2ee30f.png">
from https://mbatoul.github.io/sklearn_benchmarks/results/hpc_cluster/20211126T164327/gradient_boosting.html

The label of the y axis should be "speed-up over scikit-learn" instead "cumulated time in seconds"

**Comment by ogrisel** [comment]
you could even add "(higher is faster)"

**Comment by mbatoul** [comment]
Fixed in 0ca0a734089248de0569a991359fd295e67a5642.
